### DISCUSSION TITLE
Load new model while running templte on runpod. J´How?

### DISCUSSION THREAD

**Discussion by Codeschreibaer** [Acly/krita-ai-diffusion#423] [Q&A]
Thanks to Acly's template rendeering Ai diffuion for Krita is very convenient. Is there an easy way, to upload (or else) one special lora directly to the connected Pod from civit.ai or from my PC?. The comfy  refrrence tells me it is, but i am not sure about the storage or link matter. Thanks.  (A wildcard place for an extra lora in the template good be  fine idea, too.) yoxxx

**Comment by Acly** [comment]
There's a section at the bottom of the guide: https://github.com/Acly/krita-ai-diffusion/wiki/Cloud-GPU#custom-checkpoints-and-lora
